### ISSUE TITLE
Red Ribbom Arm Damage

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#32954]
(WEB REPORT BY: wentafk REMOTE: 206.221.180.138:7777)
# Revision
f6218eb1e6bc6707bdbab261c2bbb6cfaf0a166a

# Description
The third hand given by the red ribbon arm can take damage
# Steps to Reproduce
Acquire red ribbon artifact
Wear it
Acquire damage through lack of pressure or other means
# What you Expected
For a fake limb to not take damage
# What Actually Happened
Medical confusion

**Comment by Eneocho** [comment]
Can it heal via chems?

**Comment by Exxion** [comment]
Yeah. It's technically just a third normal arm that gets removed if you remove the item.

**Comment by Eneocho** [comment]
> Yeah. It's technically just a third normal arm that gets removed if you remove the item.

Could you amputate the third arm then replace it with a robolimb somehow?
